jekyll/_cci2/getting-started.md

```diff
@@ -15,131 +15,190 @@ This document provides a step-by-step tutorial for getting your first successful
 ## Prerequisites for Running Your First Build
 {:.no_toc}
 
-* Some basic knowledge of Git and an existing GitHub.com account or the ability to create one. This procedure uses a new GitHub repository, but CircleCI also supports the use of Bitbucket.
+* Some basic knowledge of Git.
+* A GitHub account, of which you are logged into.
+* An account on CircleCI.
 * Some basic terminal or `bash` knowledge and prior experience using the command line is helpful.
 
 ## Creating a Repository
-1. Navigate to your account on GitHub.com
-  * Go to the **Repositories** tab and then select **New** or navigate directly to [https://github.com/new](https://github.com/new). ![]( {{ site.baseurl }}/assets/img/docs/GH_Repo-New-Banner.png)
 
-2. Select Initialize this repository with a README and click the Create repository button. ![]( {{ site.baseurl }}/assets/img/docs/create-repo-circle-101-initialise-readme.png)
+Begin by creating a new repository on GitHub. You may skip this section if you intend to use an existing repository.
 
-## Adding a .yml File
+1. Navigate to GitHub and [create a new repository](https://github.com/new). 
+1. Input the name of your repository, in this case "hello-world", then click
+`Initialize this repository with a README`. Finally, click `Create repository`.
```

Comment:
I find the button clicks written as inline literals a bit confusing to look at. I think we normally just capitalise button labels? or put in " " or bold them? I'm happy with any of them as long as we're consistent 🤷‍♀ 😄

Reply:
Good point - I think we use bold elsewhere, so I'll standardize to that.